baselines/dom.generated.d.ts

```diff
@@ -3796,7 +3796,12 @@ declare var FocusNavigationEvent: {
 }
 
 interface FormData {
-    append(name: any, value: any, blobName?: string): void;
+    append(name: string, value: any, fileName?: string): void;
```

Comment:
Ah sorry, I have a quick question before I update my PR, if I can't find a FormDataEntryValue interface in the generated file does that make it ok to add it as a type anyway? Or should that be added first?